include/swift/AST/DiagnosticsSIL.def

```diff
@@ -943,7 +943,7 @@ NOTE(sil_referencebinding_inout_binding_here, none,
 //===----------------------------------------------------------------------===//
 
 NOTE(regionbasedisolation_maybe_race, none,
-     "use here could race", ())
+     "risks concurrent access", ())
```

Comment:
This sounds incomplete, and it's not obvious what risks concurrent access. Maybe `access can happen concurrently`.